planning/behavior_path_planner/autoware_behavior_path_start_planner_module/src/manager.cpp

```diff
@@ -312,8 +303,8 @@ void StartPlannerModuleManager::init(rclcpp::Node * node)
     p.th_moving_obstacle_velocity = node->declare_parameter<double>(
       surround_moving_obstacle_check_ns + "th_moving_obstacle_velocity");
     // ObjectTypesToCheck
-    std::string obj_types_ns = surround_moving_obstacle_check_ns + "object_types_to_check.";
     {
+      std::string obj_types_ns = surround_moving_obstacle_check_ns + "object_types_to_check.";
```

Comment:
Isn't it a const variable?

Reply:
I missed it this time because it was a fix for the shadowVariable warning. I will be careful from now on.